src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -127,13 +134,16 @@ def _bid(self, payload):
             return
         try:
             arguments = json.loads(payload["data"])
-            assert set(arguments.keys()) == {'price', 'energy', 'transaction_id'}
+            assert set(arguments.keys()) == {
+                'price', 'energy', 'replace_existing', 'transaction_id'}
```

Comment:
Important question here: should we always expect the `replace_existing` parameter to be provided? It will be set to `True` by default on the api client (so it will always be provided from that source) but let me know if I should make it optional here and omit this strict check.

Reply:
I think it is better to make it optional